SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -292,6 +294,19 @@ Interest InterestFactory::CreateServerWorkerInterest()
 	ComponentInterest Queries;
 	Queries.Queries.Add(Query);
 
+	if (SpatialGDKSettings->bEnableUnrealLoadBalancer && SpatialGDKSettings->bEnableServerQBI)
+	{
+		if (const UAbstractLBStrategy* LoadBalancer = NetDriver.LoadBalanceStrategy)
+		{
+			// The load balancer won't be ready when the worker initially connects to SpatialOS. It needs
+			// to wait for the virtual worker mappings to be replicated.
+			if (LoadBalancer->IsReady())
+			{
+				LoadBalancer->CreateWorkerInterestQueries(Queries.Queries);
+			}
```

Comment:
Chat to me when you revisit this, we should be able to guarantee when it's available in a specific place vs. just jamming into addcomponent / component update callbacks